arviz/stats/diagnostics.py

```diff
@@ -813,13 +856,48 @@ def _mcse_mean(ary):
         return np.nan
     ess = _ess_mean(ary)
     if _numba_flag:
-        sd = _sqrt(svar(np.ravel(ary), ddof=1), np.zeros(1))
+        sd = _sqrt(svar(np.ravel(ary), ddof=1), 0)
     else:
         sd = np.std(ary, ddof=1)
     mcse_mean_value = sd / np.sqrt(ess)
     return mcse_mean_value
 
 
+def _mcse_func_sbm(ary, func, b=None, var_func=np.var, func_kwargs=None):
+    """Compute the Markov Chain error on an arbitrary function."""
+    ary = np.asarray(ary)
+    if _not_valid(ary, shape_kwargs=dict(min_draws=10, min_chains=1)):
+        return np.nan
+    n = ary.size
+    if b is None:
+        b = int(np.sqrt(n))
+    if func_kwargs is None:
+        func_kwargs = {}
+    func_estimate_sd = _sbm(ary, func, b=b, var_func=var_func, func_kwargs=func_kwargs)
+    mcse_func_value = func_estimate_sd / np.sqrt(n)
+    return mcse_func_value
+
+
+def _sbm(ary, func, b, var_func, func_kwargs):
+    """Subsampling bootstrap method.
+
+    References
+    ----------
+    .. [1] Doss, Charles R., et al. "Markov chain Monte Carlo estimation of quantiles."
+       *Electronic Journal of Statistics* 8.2 (2014): 2448-2478.
+       https://doi.org/10.1214/14-EJS957
+
+    """
+    flat_ary = np.ravel(ary)
+    n = len(flat_ary)
+    func_estimates = np.empty(n - b)
+    for i in range(n - b):
+        sub_ary = flat_ary[i : i + b]
+        func_estimates[i] = func(sub_ary, **func_kwargs)
+    func_estimate_sd = np.sqrt(b * var_func(func_estimates))
```

Comment:
we should probably decide API-wise if we want to keep this or instead move to `std_func` and multiply that by the square root of `b`.